src/libraries/System.Private.CoreLib/src/System/Decimal.DecCalc.cs

```diff
@@ -184,20 +184,7 @@ private static ulong UInt32x32To64(uint a, uint b)
 
             private static void UInt64x64To128(ulong a, ulong b, ref DecCalc result)
```

Comment:
This can be replaced, technically speaking, by `Math.BigMul(ulong, ulong, out ulong)` as well.

Then it can also be deleted.

Reply:
Yeah sure, NB that overflow check and insertion to DecCalc result are still in body below. Do we prefer to move those around to where this is now called or should i keep this method?

Reply:
Probably better to keep the extra checks centralized here and just defer the algorithm to Math.BigMul